src/Maybe/find.js

```diff
@@ -0,0 +1,30 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Dale Francis (dalefrancis88) */
+
+const curry = require('../core/curry')
+const Maybe = require('./index')
```

Comment:
This code is compiled, so you we can safely remove the need for name spacing and use desturcutoring like:
```js
const { Nothing, Just } = require('.')
```